client/src/app/pages/applications/analysis-wizard/set-mode.tsx

```diff
@@ -71,7 +71,9 @@ export const SetMode: React.FC<ISetMode> = ({ isSingleApp, isModeValid }) => {
             toggleAriaLabel="Analysis mode dropdown toggle"
             aria-label={name}
             value={value}
-            onChange={onChange}
+            onChange={(value) => {
```

Comment:
this change is only reformatting - I would keep the original format as it seems less verbose.

Reply:
I returned to the previous state

Reply:
it seems the changes were not pushed yet

Reply:
I uploaded the changes